pkg/chunk/table_manager.go

```diff
@@ -47,39 +47,39 @@ func init() {
 // TableManagerConfig holds config for a TableManager
 type TableManagerConfig struct {
 	// Master 'off-switch' for table capacity updates, e.g. when troubleshooting
-	ThroughputUpdatesDisabled bool
+	ThroughputUpdatesDisabled bool `yaml:"throughput_updates_disabled"`
 
 	// Master 'on-switch' for table retention deletions
-	RetentionDeletesEnabled bool
+	RetentionDeletesEnabled bool `yaml:"retention_deletes_enabled"`
 
 	// How far back tables will be kept before they are deleted
-	RetentionPeriod time.Duration
+	RetentionPeriod time.Duration `yaml:"retention_period"`
 
 	// Period with which the table manager will poll for tables.
-	DynamoDBPollInterval time.Duration
+	DynamoDBPollInterval time.Duration `yaml:"dynamodb_poll_interval"`
 
 	// duration a table will be created before it is needed.
-	CreationGracePeriod time.Duration
+	CreationGracePeriod time.Duration `yaml:"creation_grace_period"`
 
-	IndexTables ProvisionConfig
-	ChunkTables ProvisionConfig
+	IndexTables ProvisionConfig `yaml:"index_tables_provisioning"`
+	ChunkTables ProvisionConfig `yaml:"chunk_tables_provisioning"`
 }
 
 // ProvisionConfig holds config for provisioning capacity (on DynamoDB)
 type ProvisionConfig struct {
-	ProvisionedThroughputOnDemandMode bool
-	ProvisionedWriteThroughput        int64
-	ProvisionedReadThroughput         int64
-	InactiveThroughputOnDemandMode    bool
-	InactiveWriteThroughput           int64
-	InactiveReadThroughput            int64
-
-	WriteScale              AutoScalingConfig
-	InactiveWriteScale      AutoScalingConfig
-	InactiveWriteScaleLastN int64
-	ReadScale               AutoScalingConfig
-	InactiveReadScale       AutoScalingConfig
-	InactiveReadScaleLastN  int64
+	ProvisionedThroughputOnDemandMode bool  `yaml:"provisioned_throughput_on_demand_mode"`
+	ProvisionedWriteThroughput        int64 `yaml:"provisioned_write_throughput"`
+	ProvisionedReadThroughput         int64 `yaml:"provisioned_read_throughput"`
+	InactiveThroughputOnDemandMode    bool  `yaml:"inactive_throughput_on_demand_mode"`
+	InactiveWriteThroughput           int64 `yaml:"inactive_write_throughput"`
+	InactiveReadThroughput            int64 `yaml:"inactive_read_throughput"`
+
+	WriteScale              AutoScalingConfig `yaml:"write_scale"`
+	InactiveWriteScale      AutoScalingConfig `yaml:"inactive_write_scale"`
+	InactiveWriteScaleLastN int64             `yaml:"inactive_write_scale_lastn"`
```

Comment:
Maybe add an underscore before the "n". This wayit would be consistent delimitation with the camelcase of the variable names